app/(storyblok)/[...slug]/page.tsx

```diff
@@ -107,11 +103,12 @@ async function getStoryData(params: { slug: string[] }) {
         }
       }
       catch (e) {
-        throw error;
+        console.error('Error', error);
       }
     }
-    throw error;
   }
+
+  return { data: 404 };
```

Comment:
Return a 404 if the page doesn't get data from Storyblok and handle the 404 error on the server component response.